src/main/java/org/candlepin/subscriptions/tally/MetricUsageCollector.java

```diff
@@ -413,36 +388,43 @@ private String getCloudProviderAsString(Event.CloudProvider cloudProvider) {
   }
 
   private BillingProvider getBillingProvider(Event.BillingProvider billingProvider) {
-    switch (billingProvider) {
-      case __EMPTY__:
-        return null;
-      case AWS:
-        return BillingProvider.AWS;
-      case AZURE:
-        return BillingProvider.AZURE;
-      case ORACLE:
-        return BillingProvider.ORACLE;
-      case GCP:
-        return BillingProvider.GCP;
-      case RED_HAT:
-        return BillingProvider.RED_HAT;
-      default:
-        throw new IllegalArgumentException(
-            String.format("Unsupported value for billing provider: %s", billingProvider.value()));
-    }
+    return switch (billingProvider) {
+      case __EMPTY__ -> null;
+      case AWS -> BillingProvider.AWS;
+      case AZURE -> BillingProvider.AZURE;
+      case ORACLE -> BillingProvider.ORACLE;
+      case GCP -> BillingProvider.GCP;
+      case RED_HAT -> BillingProvider.RED_HAT;
+    };
   }
 
-  private void addBucketsFromEvent(Host host, Event event, Optional<TagMetaData> serviceTypeMeta) {
+  private void addBucketsFromEvent(Host host, Event event, String serviceType) {
+    // We have multiple SubscriptionDefinitions that can have the same serviceType (OpenShift
+    // Cluster).  The SLA and usage for these definitions should be the same (if defined), so we
+    // need to collect them all, deduplicate, and then verify via MoreCollectors.toOptional that we
+    // have only one or zero choices.
+    var subDefinitions = SubscriptionDefinition.findByServiceType(serviceType);
+    Optional<String> sla =
+        subDefinitions.stream()
+            .map(x -> x.getDefaults().getSla().toString())
+            .distinct()
+            .collect(MoreCollectors.toOptional());
+    Optional<String> usage =
+        subDefinitions.stream()
+            .map(x -> x.getDefaults().getUsage().toString())
+            .distinct()
+            .collect(MoreCollectors.toOptional());
+
     ServiceLevel effectiveSla =
         Optional.ofNullable(event.getSla())
             .map(Event.Sla::toString)
             .map(ServiceLevel::fromString)
-            .orElse(serviceTypeMeta.map(TagMetaData::getDefaultSla).orElse(ServiceLevel.EMPTY));
+            .orElse(sla.map(ServiceLevel::fromString).orElse(ServiceLevel.EMPTY));
     Usage effectiveUsage =
         Optional.ofNullable(event.getUsage())
             .map(Event.Usage::toString)
             .map(Usage::fromString)
-            .orElse(serviceTypeMeta.map(TagMetaData::getDefaultUsage).orElse(Usage.EMPTY));
+            .orElse(usage.map(Usage::fromString).orElse(Usage.EMPTY));
```

Comment:
For better encapsulation, please move this to a static method on SubscriptionDefinition named `defaultUsageForSubscriptionDefinitions`